CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- New handler that is setting `MachinePool` `ReplicasReady` condition. It uses almost same logic to set the condition as the upstream Cluster API controller.
```

Comment:
> It uses almost same logic to set the condition as the upstream Cluster API controller

This sentence confuses me. It doesn't help me understand what this new handler does, so it doesn't make sente to add it to the changelog IMHO.

Reply:
It's to state that we are implementing it similarly/compatibly to upstream, which is important in the context of moving to upstream controller, as the behaviour will not break, but it's probably enough to have it in the pull request description. Updated changelog.